http-clients/src/main/java/org/triplea/http/client/JsonEncoder.java

```diff
@@ -15,6 +15,7 @@
 public class JsonEncoder implements Encoder {
   private static final GsonEncoder gsonEncoder = new GsonEncoder();
 
+  @SuppressWarnings("deprecation")
   @Override
   public void encode(final Object object, final Type bodyType, final RequestTemplate template) {
     if (bodyType.getTypeName().equals(String.class.getName())) {
```

Comment:
Just looked up the source:
https://github.com/OpenFeign/feign/blob/23ee09ef88a02ead7c8c925cdab7b094053b3d9e/core/src/main/java/feign/RequestTemplate.java#L754
It seems like creating a `Request.Body` instance and using that is the preffered way now.

Also the default code uses reference equality i.e. `bodyType == String.class` https://github.com/OpenFeign/feign/blob/23ee09ef88a02ead7c8c925cdab7b094053b3d9e/core/src/main/java/feign/codec/Encoder.java#L88

not sure if you changed that to meet a errorprone rule or something.